src/openloco/windows/IndustryWindow.cpp

```diff
@@ -0,0 +1,1064 @@
+        {
+            switch (widgetIndex)
+            {
+                case common::widx::caption:
+                    common::renameIndustryPrompt(self, widgetIndex);
+                    break;
+
+                case common::widx::close_button:
+                    WindowManager::close(self);
+                    break;
+
+                case common::widx::tab_industry:
+                case common::widx::tab_production:
+                case common::widx::tab_production_2:
+                case common::widx::tab_transported:
+                    common::switchTab(self, widgetIndex);
+                    break;
+            }
+        }
+
+        // 0x004565FF
+        static void on_resize(window* self)
+        {
+            {
+                self->set_size(minWindowSize, maxWindowSize);
+            }
+        }
+
+        static void initEvents()
+        {
+            events.draw = draw;
+            events.on_mouse_up = on_mouse_up;
+            events.on_resize = on_resize;
+            events.on_update = common::update;
+            events.prepare_draw = prepare_draw;
+            events.text_input = common::text_input;
+        }
+    }
+
+    namespace transported
+    {
+        static const gfx::ui_size_t windowSize = { 300, 127 };
+
+        static widget_t widgets[] = {
+            commonWidgets(300, 126, string_ids::title_statistics),
+            widget_end(),
+        };
+
+        static window_event_list events;
+
+        // 0x00456665
+        static void prepare_draw(window* self)
+        {
+            common::prepare_draw(self);
+
+            common::repositionTabs(self);
+        }
+
+        // 0x00456705
+        static void draw(window* self, gfx::drawpixelinfo_t* dpi)
+        {
+            self->draw(dpi);
+            common::drawTabs(self, dpi);
+
+            auto industry = industrymgr::get(self->number);
+            auto industryObj = objectmgr::get<industry_object>(industry->object_id);
+            auto totalReceivedCargo = industryObj->received_cargo[0] & industryObj->received_cargo[1] & industryObj->received_cargo[2];
+            int16_t xPos = self->x + 3;
+            int16_t yPos = self->y + 45;
+            gfx::point_t origin = { xPos, yPos };
+            // Draw Last Months received cargo stats
+            if (totalReceivedCargo != 0xFF)
+            {
+                origin.x += 4;
+                origin.y += 10;
+                gfx::draw_string_494B3F(*dpi, xPos, yPos, colour::black, string_ids::received_cargo);
+                for (uint8_t i = 0; i < std::size(industryObj->received_cargo); i++)
+                {
+                    uint16_t receivedCargo = industryObj->received_cargo[i];
+                    if (receivedCargo != 0xFF)
+                    {
+                        auto cargoObj = objectmgr::get<cargo_object>(receivedCargo);
+                        auto args = FormatArguments();
+
+                        if (receivedCargo == 1)
```

Comment:
This is why it is important to name you're variables better. This is the `receivedCargoType` not the `receviedCargoQuantity` this line does not make sense. You will need to use the variable from L703.